src/components/Menu/MenuItem.tsx

```diff
@@ -97,6 +97,19 @@ export function MenuItem({
 
   const currentStyle = current ? 'menu__list-item__link--current' : '';
 
+  let hideAPIResources = false; 
+
+  if ((currentPlatform == 'javascript' || 
```

Comment:
I think we have a list of web platforms we could use here

Reply:
Yes, yes there is. Fixed, thanks!